core/storage/user/gae_models.py

```diff
@@ -110,6 +110,22 @@ class UserSettingsModel(base_models.BaseModel):
     preferred_audio_language_code = ndb.StringProperty(
         default=None, choices=[
             language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES])
+    # Displayed name for users on android. They are editable unlike usernames.
+    name = ndb.StringProperty(default=None)
+    # A code associated with profiles to provide a PIN based authentication
```

Comment:
Should this be part of ``UserAuthModel``, instead?

Reply:
I don't think so because it does not correspond to any authentication related information. It's just a user related information like user bio. And most importantly we can uniquely identify a person by their `name` because its meant to be unique only within the account, not across all users.

Reply:
I too think that this should be part of `UserAuthModel`, also should the PIN be encrypted somehow?

Reply:
Sorry I am unable to understand this discussion. I was talking about `name` attribute, were both of you referring to `PIN` instead this whole time? 
If it's `PIN` then pls refer to this [comment](https://github.com/oppia/oppia/pull/10105#discussion_r463133394). If not, then please tell me more about the problem do you see.

Reply:
The comment was on line 115 which corresponds to the PIN. I'll follow up on that discussion thread.

Reply:
Done. Shifted `PIN` to `UserAuthModel` now in PR #10004.